src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -138,17 +149,28 @@ class AddressValidationModal extends React.Component {
     const showEditLinkErrorState = addressValidationError && validationKey;
     const showEditLinkNonErrorState = !addressValidationError;
     const showEditLink = showEditLinkErrorState || showEditLinkNonErrorState;
+    const showFirstRadioButton =
+      (isAddressFromUser && validationKey) || !isAddressFromUser;
 
     return (
-      <div key={id}>
-        <input
-          style={{ zIndex: '1' }}
-          type="radio"
-          name={id}
-          disabled={isAddressFromUser && !validationKey}
-          checked={selectedId === id}
-          onClick={this.onChangeHandler(address, id)}
-        />
+      <div
+        onClick={this.onChangeHandler(address, id)}
```

Comment:
it doesn't need to have the onClick handler if the radio button isn't visible, right?

Reply:
Fixed